src/Autocomplete/doc/index.rst

```diff
@@ -250,7 +250,7 @@ to the options above, you can also pass:
 
 ``filter_query`` (default: ``null``)
     If you want to completely control the query made for the "search results",
-    use this option. This is incompatible with ``searchable_fields``::
+    use this option. This is incompatible with ``searchable_fields`` and ``max_results``::
```

Comment:
Nice catch here!

Would you also update the BaseAutocompleteType code to make the same check than filter/searcheable_fields ?  https://github.com/symfony/ux/blob/eab7db5c90401d943cdb64394d01ed1579cc9694/src/Autocomplete/src/Form/BaseEntityAutocompleteType.php#L68-L75

Reply:
Can be a problem because max_results has default to 10.